src/cli.rs

```diff
@@ -9,17 +9,17 @@ use std::path::PathBuf;
 /// Any whitespace is ignored.
 #[bpaf(options, version)]
 pub struct Config {
-    /// Location of puzzle to read.
+    /// Location of puzzle to read or stdin by default
     #[bpaf(short, long, argument("FILE"))]
     pub input: Option<PathBuf>,
-    /// Output file to write solution to. Leave blank to write to stdout.
+    /// Output file to write solution to or stdout by default
     #[bpaf(short, long, argument("FILE"))]
     pub output: Option<PathBuf>,
     /// Print puzzle with nice borders, options include `simple`, `multiline` and `bordered`
-    #[bpaf(long, argument("STYLE"), fallback(OutputStyle::Bordered))]
+    #[bpaf(long, argument::<FromUtf8<OutputStyle>>("STYLE"), fallback(OutputStyle::Bordered))]
```

Comment:
I wanted to use `FromStr` but didn't realize you could add this to the macro. Thanks a lot for this fix.